test/integration/client/testdata/exec-plugin.sh

```diff
@@ -23,4 +23,6 @@ if [[ -n "${EXEC_PLUGIN_OUTPUT_FILE-""}" ]]; then
   exit
 fi
 
-echo "$EXEC_PLUGIN_OUTPUT"
+echo "${EXEC_PLUGIN_OUTPUT-""}"
+
+exit "${EXEC_PLUGIN_EXEC_CODE-0}"
```

Comment:
nit: fix missing newline

Reply:
Included in 31c53024ff9c54597a7c3da511eb5e864f5e6140.